grype/presenter/json/test-fixtures/snapshot/TestJsonDirsPresenter.golden

```diff
@@ -100,72 +102,42 @@
    ],
    "artifact": {
     "name": "package-1",
-    "version": "1.0.1",
+    "version": "1.1.1",
     "type": "deb",
     "locations": [
      {
-      "path": "/some/path/pkg1"
+      "path": "/somefile-1.txt"
      }
     ],
     "language": "",
     "licenses": [],
-    "cpes": [],
-    "purl": "",
-    "upstreams": []
-   }
-  },
-  {
```

Comment:
double-check lost vulnerability here since we want both packages and vulns in the snapshots

Reply:
removed in favor of match 1 ==> p1 && match 2 ==> p2